packages/meteor-accounts-saml/saml_rocketchat.coffee

```diff
@@ -71,14 +84,21 @@ getSamlConfigs = (service) ->
 # Meteor saml package uses Accounts.saml.settings.
 
 configureSamlService = (samlConfigs) ->
-	try
-		privateKey = fs.readFileSync(samlConfigs.privateKeyFilePath).toString()
-		privateCert = fs.readFileSync(samlConfigs.publicCertFilePath).toString()
-	catch err
-		console.warn "Can't configure key or cert for SAML, signing will not be performed by RocketChat"
-		console.warn "Error was: #{err.message}"
-		privateKey = false
-		privateCert = false
+	privateKey = false
+	privateCert = false
+	if samlConfigs.privateKeyFilePath and samlConfigs.publicCertFilePath
+		try
+			privateKey = fs.readFileSync(samlConfigs.privateKeyFilePath).toString()
+			privateCert = fs.readFileSync(samlConfigs.publicCertFilePath).toString()
+			logger.info "Successfully loaded public cert and private key files for SAML."
+		catch err
+			logger.error "Can't configure key or cert for SAML, signing will not be performed by RocketChat."
+			logger.error "Error was: #{err.message}"
+	else
+		if samlConfigs.privateKeyFilePath or samlConfigs.publicCertFilePath
```

Comment:
ugh - I kind of want it to be both or nothing.  I'm using logger not console now too.